m4_sessions/1811_inning/timprov.md

```diff
@@ -26,6 +34,11 @@
 
 ## Session Outline (Agenda for session, include timing for each session and who is facilitating)
 
+Intro
```

Comment:
Maybe add some "norms" so that people are really engaging and creating a positive space

Reply:
Agreed. I will add this